FAQ.md

```diff
@@ -67,6 +67,14 @@ If you still want to enable this functionality, run `ujust toggle-ghns`
 
 Xwayland is disabled by default on GNOME, KDE Plasma, and Sway. Use `ujust toggle-xwayland` if you need it
 
+#### Why I can't install & use any Gnome extensions?
+
+This is because support for them has been intentionally disabled in secureblue.
+Only system extensions are trusted, if they are installed.
+
+To enable support for Gnome extensions, you can run ujust command:
+`ujust toggle-gnome-extensions enable`
```

Comment:
this is a toggle, so specifying "enable" should not be required. Compare to the other toggles for the pattern

Reply:
Alright, I will simplify the just command